README.md

```diff
@@ -1,2 +1,41 @@
-# go-ipmctl
+# Go language bindings for the ipmctl API for managing PMMs
+
 Go bindings for the libipmctl native C lib to enable management of DCPMMs from Go source 
+
+This is a Go interface for [ipmctl](https://github.com/intel/ipmctl) which is also a work in progress.
+
+## How to Build
+
+This is a [Go](https://golang.orghttps://golang.org/doc/install)
+project, so the Go development tools are naturally required. We
+recommend the most current Go release available. As of July 2018, the project has been built and tested with Go 1.9.
```

Comment:
we could update this to go 1.11.5 which is what we are currently testing in CI

Reply:
Agreed. Will change.